src/rules/custom-property-empty-line-before/__tests__/index.js

```diff
@@ -0,0 +1,147 @@
+import {
+  mergeTestDescriptions,
+  testRule,
+} from "../../../testUtils"
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+
+const rule = rules[ruleName]
+
+const alwaysTests = {
+  accept: [ {
+    code: "a{\n\n --custom-prop: value; }",
+    description: "basic rule",
+  }, {
+    code: "a{\n\n --custom-prop: value;\r\n\r\n --custom-prop2: value;}",
+  }, {
+    code: "a{\n\n --custom-prop: value;\n\r\n --custom-prop2: value;}",
```

Comment:
Can we get an `accept` and `reject` test for a custom property that follows:

1. A declaration e.g. `top: 10px;`
2. A nested at-rule. `@extends .class;`
3. A dollar-variable. `$var: value`